.github/workflows/checks.yaml

```diff
@@ -0,0 +1,42 @@
+name: Check code
+
+env:
+  DOCKER_BUILDKIT: 1
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'feature/**'
+      - 'bugfix/**'
+      - 'hotfix/**'
+      - 'develop'
+
+jobs:
+  check_web:
+    name: Check Python
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v3
+      with:
```

Comment:
https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python#using-multiple-python-versions

need to test py 3.9, 10, 11 (and 12 once its properly on github actions)